doc/ivoatexmeta.tex

```diff
@@ -1,6 +1,7 @@
 % GENERATED FILE -- edit this in the Makefile
 \newcommand{\ivoaDocversion}{1.0}
-\newcommand{\ivoaDocdate}{2022-11-04}
-\newcommand{\ivoaDocdatecode}{20221104}
+\newcommand{\ivoaDocdate}{2022-11-11}
+\newcommand{\ivoaDocdatecode}{20221111}
```

Comment:
Please don't have ivoatexmeta in the repository.  It will only cause unnecessary edit conflicts and diff noise.

Reply:
file removed from the repo